gateway/1.6/modules/ROOT/pages/flex-troubleshoot-soap-apis.adoc

```diff
@@ -0,0 +1,29 @@
+= Publishing SOAP APIs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway does not natively support SOAP APIs and does not provide any schema validation for XML. However, you can publish an HTTP API instance to a secure SOAP upstream service.
+
+To create an HTTP API instance to a secure SOAP API upstream service, refer to the relevant tutorial for your running mode:
```

Comment:
```suggestion
To create an HTTP API instance to a secure SOAP API upstream service, refer to the following:
```
Since the xlink'd pages have the run mode in the title, I'm thinking it's clear that the tutorials are run-mode-specific.